src/actions/GroupActions.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
it feels a bit unfortuate that the .js files have been replaced with .ts files rather than renamed to .ts files, thus losing commit history...

Reply:
...and making this really hard to review

Reply:
This is actually on the dispatcher PR, and git decided that it doesn't want to handle it as a rename.